internal/e2e/rollup_code_splitting/BUILD.bazel

```diff
@@ -11,18 +11,32 @@ rollup_bundle(
     license_banner = ":license.txt",
 )
 
+rollup_bundle(
```

Comment:
hmm, I think we want to just change most of the tests to remove addl_entry_points and test the auto behavior everywhere, except maybe for one test that's left on the legacy behavior just to make sure we don't break it during the transition.

Reply:
Ok, here's another branch where I just convert the test to auto, deleting test coverage for (non auto) additional entry points. I have to delete the tests for additional_entry.js to get this to work. A new chunk-X.js is created (it's working) but I think I can only write a test for it if put X (the hash) into the test. Do you think that's OK? 

https://github.com/bazelbuild/rules_nodejs/compare/master...Jesse-Good:i825-autoEntryPoints2?expand=1

Reply:
PTAL. I've updated this branch.

Reply:
Ok, I've updated :bundle to enable code splitting. And I added a new test to cover the previous behavior with specifying additional_entry_points.